lib/filter.g

```diff
@@ -344,4 +345,70 @@ BIND_GLOBAL( "CANONICAL_BASIS_FLAGS", QUO_INT(SUM_FLAGS,5) );
 
 #############################################################################
 ##
+#V  IdOfFilterByName
+#V  FilterByName
+##
+##  <#GAPDoc Label="FilterByName">
+##  <ManSection>
+##  <Func Name="IdOfFilterByName" Arg="name"/>
+##  <Func Name="FilterByName" Arg="name"/>
+##
+##  <Description>
+##  finds the id of the filter, or the filter with name <A>name</A> respectively,
+##  in the global FILTERS list. 
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+BIND_GLOBAL( "IdOfFilterByName",
+function(name)
+    local fid;
+    for fid in [1..LEN_LIST(FILTERS)] do
+        if NAME_FUNC(FILTERS[fid]) = name then
+           return fid;
+        fi;
+    od;
+    return fail;
+end);
```

Comment:
So this function is essentially equivalent to `name -> PositionProperty(FILTERS, f -> NAME_FUNC(f) = name)` ? Of course `PositionProperty` is not available at the time this file is read. But it makes me wonder how useful this function *really* is... seem to me mostly something for core developers? Perhaps you are concerned about this helper being available very early on? Because if not, then I'd consider going with the much shorter definition, and simply defining this in a new filter `lib/filter.gi`